### PR TITLE
chore: address a few typos

### PR DESCRIPTION
**Pull request by radu-matei**
Signed-off-by: Radu Matei <radu.matei@fermyon.com>

### CODE REVIEW

File: design/high-level/Choices.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Based on the [goals](Goals.md) and [use cases](UseCases.md), the component
 model makes several high-level design choices that permeate the rest of the
 component model.
 
-1. The component model adopts a shared-nothing architecture in which component
+1. The component model adopts a shared-nothing architecture in which components
```
Comment (lukewagner) [inline design/high-level/Choices.md:7]:
```suggestion
1. The component model adopts a shared-nothing architecture in which component
```

Comment (lukewagner) [reply design/high-level/Choices.md:7]: I think this one was fine before; the intended phrase is indeed "component instances" (i.e., the plural of "component instance", which refers to a (dynamic, stateful) instance of a (static, stateless) component).

**Review by lukewagner — APPROVED** [review]
Thanks!

File: design/high-level/Choices.md (hunk 1 above)
Comment (radu-matei) [reply design/high-level/Choices.md:7]: Ah, you are right.
Reverting.

**Comment by lukewagner** [comment]
Thanks again!

**Pull request merged**